site/content/contribute/getting-started/code-review.md

```diff
@@ -85,7 +85,8 @@ If you are a core committer seeking a review
     * Merge your pull request and delete the branch if not from a fork.
         - Note that any core committer is free to merge on your behalf.
         - If your pull request depends on other pull requests, consider assigning the `Do Not Merge/Awaiting PR` label.
-9. For handling cherry-picks, please check [here](https://developers.mattermost.com/contribute/getting-started/branching/).
```

Comment:
While the tick-tock strategy is old, it feels like https://developers.mattermost.com/contribute/getting-started/branching/#cherry-pick-process---developer has a lot of good detail around how do do cherry picking especially when the automation fails (because of merge conflicts).

Should we find a way to keep that and continue linking there?

Reply:
Sounds good!